.github/workflows/build-docs.yml

```diff
@@ -126,22 +126,30 @@ jobs:
           mkdir -p artifacts/docs
           mv cuda_python/docs/build/html/* artifacts/docs/
 
-      # Note: currently this is only for manual inspection. This step will become
-      # required once we switch to use GHA for doc deployment (see the bottom).
+          # create an empty folder for removal use
+          mkdir -p artifacts/empty_docs
+
+      # TODO: Consider removing this step?
       - name: Upload doc artifacts
         uses: actions/upload-pages-artifact@v3
         with:
           path: artifacts/
           retention-days: 3
 
-      # The step below is not executed unless when building on main.
+      - name: Deploy or clean up doc preview
+        uses: ./.github/actions/doc_preview
+        continue-on-error: false
```

Comment:
This is the default. I think it's better to omit this line (otherwise someone might think it's special somehow).

Reply:
done in 026df62